e2e-pw/scripts/generate-screenshots-docker-image/Dockerfile

```
@@ -13,13 +13,15 @@ RUN apt-get update && apt-get install -y software-properties-common \
+    /uv /uvx /usr/local/bin/
 
-RUN python -m venv /e2e/venv \
-    && . /e2e/venv/bin/activate \
-    && cd /e2e/fiftyone \
-    && pip install --upgrade pip \
-    && pip install -r requirements/e2e.txt \
-    && pip install -e .
+ENV UV_PROJECT_ENVIRONMENT=/e2e/venv
+ENV PATH="/e2e/venv/bin:${PATH}"
+
+RUN cd /e2e/fiftyone \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)